var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -40,7 +41,13 @@ class Pfunit(CMakePackage):
 
     depends_on('python@2.7:', type=('build', 'run'))  # python3 too!
     depends_on('mpi', when='+mpi')
+    depends_on('m4', when='@4.1.5:4.1.9')
```

Comment:
It's likely that since 4.1.9 is the latest, future requirements will also include 4.1.9. I'd change this to `@4.1.5:`.